map_gen/maps/crash_site/outpost_builder.lua

```diff
@@ -1213,6 +1219,31 @@ local function do_artillery_turrets_targets()
     end
 end
 
+local function set_pollution_multiplier(args, player)
+    game.print(args.multiplier)
+
+    local multipliers = {}
+    for m in string.gmatch(args.multiplier, "%-?%d+") do -- Assuming the surface name isn't a valid number.
+        table.insert(multipliers, tonumber(m))
+    end
+    if not multipliers or #multipliers ~= 1 then
+        game.player.print("Fail")
+        return
+    end
```

Comment:
Are you planning on having multiple multipliers?
Can this just be
```suggestion
    local multiplier = tonumber(args.multiplier)
    if not multiplier then
        player.print("Fail")
        return
    end   
```